### PR TITLE
Add script EN-ID replacer to CI

### PR DESCRIPTION
**Pull request by wahyuoi**
@girikuncoro

### CODE REVIEW

File: scripts/localize_url.py [hunk 1]
```diff
@@ -11,19 +11,33 @@ def get_files(path):
...
+  found = False
   for k,v in bad_links.items():
     if len(v) > 0:
       print(k)
       for vv in v:
         print("--> " + vv)
+        found = True
       print()
+  if (found == False):  
+    print("ok, no need to replace anything")
```
Comment (girikuncoro) [inline scripts/localize_url.py:23]: you can actually simply add this simple check in line 14 and return early, len(bad_links) will give the total keys existed in a dict
i.e.
```
if (len(bad_links)) == 0:
  print("ok, no need to replace anything")
  return
```

**Comment by girikuncoro** [comment]
@wahyuoi let's see the CI jobs

**Comment by girikuncoro** [comment]
nice, it works, thanks for doing this! https://app.circleci.com/pipelines/github/jk8s/sig-docs-id/33/workflows/07e5f75b-a37e-456d-bb62-11f629731694/jobs/57/parallel-runs/0/steps/0-109

we do have many EN links 🤔

**Comment by wahyuoi** [comment]
@girikuncoro  https://github.com/kubernetes/website/pull/22461 

we need to merge this PR to resolve them

**Comment by girikuncoro** [comment]
This got merged @wahyuoi

**Comment by wahyuoi** [comment]
ah sorry, i mean my PR on k/website #22461

**Comment by girikuncoro** [comment]
Ah sorry I saw this on my mobile phone. Cool I just merged it, great work on the script!

**Pull request merged**